bindings/go/evmc/evmc.go

```diff
@@ -110,6 +110,12 @@ func (err Error) Error() string {
 		return "evmc: precompile failure"
 	case C.EVMC_CONTRACT_VALIDATION_FAILURE:
 		return "evmc: contract validation failure"
+	case C.EVMC_ARGUMENT_OUT_RANGE:
```

Comment:
Should be `EVMC_ARGUMENT_OUT_OF_RANGE`. See Go CI builds.

Reply:
Fixed already, just didn't push before the other one is decided.